libs/commons/UTmp.ml

```diff
@@ -98,7 +98,8 @@ end
 (* API using Fpath.t for filenames *)
 (*****************************************************************************)
 
-let get_temp_dir_name () = Fpath.v (UFilename.get_temp_dir_name ())
+let get_temp_dir_name () =
+  Fpath.v (UFilename.get_temp_dir_name () |> UUnix.realpath)
```

Comment:
This is not desirable in general and it should not be needed since `Testutil.mask_temp_paths ()` masks both variants of the temp dir path.

Reply:
No, I cannot fix unspecified tests that fail on a platform I don't have access to.

Reply:
I understand, but if you also reject PRs of people who have access to the platform that fix the issue, it complicate things.

Reply:
Anyway hopefully someone will fix it properly.

Reply:
@mjambon you have access via CI in PR #10096. You can push changes to that branch if you want to test whether they fix the failures.